packages/injectors-react/src/makeHook.test.js

```diff
@@ -41,4 +49,46 @@ describe('makeHook', () => {
 		expect(store.injectThings).toHaveBeenCalledTimes(1);
 		expect(store.injectThings.mock.calls[0][0]).toEqual({ foo: noop }, { namespace: 'yolo' });
 	});
+
+	it('throws warning if useNamespace is provided, but isGlobal flag is not', () => {
+		const warn = jest.spyOn(global.console, 'warn');
+
+		mount(
+			<NamespaceProvider store={store} useNamespace={always('yolo')}>
+				<Test>{() => useThings({ foo: noop })}</Test>
+			</NamespaceProvider>
+		);
+
+		expect(warn).toHaveBeenCalledWith(...consoleWarningContent);
+	});
+
+	it('does not throw warning if isGlobal is not provided', () => {
+		const warn = jest.spyOn(global.console, 'warn');
+
+		mount(
+			<NamespaceProvider store={store}>
+				<Test>{() => useThings({ foo: noop })}</Test>
+			</NamespaceProvider>
+		);
+
+		expect(warn).toHaveBeenCalledTimes(0);
+	});
+
```

Comment:
please add test for `"does not warn if namespace is passed and isGlobal is not passed"` and some more combinations if you can think of them.